helm_deploy/values-dev.yaml

```diff
@@ -31,7 +31,7 @@ env:
   HMPPS_AUTH_BASE_URI: "https://sign-in-dev.hmpps.service.justice.gov.uk/auth"
   HMPPS_AUTH_REDIRECT_BASE_URI: "https://calculate-journey-variable-payments-dev.apps.live-1.cloud-platform.service.justice.gov.uk"
   CRON_AUTOMATIC_LOCATION_MAPPING: "0 30 13 * * ?"
-  CRON_IMPORT_REPORTS: "0 30 5 * * ?"
+  CRON_IMPORT_REPORTS: "-"
```

Comment:
I don't want this to run.  I have applied the cron job to run at 9am in dev and will be monitoring to make sure it runs as planned.